panel/models/datetime_picker.py

```diff
@@ -10,7 +10,7 @@ class DatetimePicker(InputWidget):
 
     '''
 
-    value = Nullable(String, help="""
+    value = Nullable(Either(String, Date, Datetime), help="""
```

Comment:
Is this really needed or correct? What does bokeh convert the Date, Datetime values to?